webapp/application.py

```diff
@@ -250,7 +250,11 @@ def _get_application(application_id):
             except HTTPError as error:
                 print(error)
 
-            if (
+            if job_id == "2680006":  # Enterprise Sales Representative
```

Comment:
What's the chance of this ID changing in the future? Are we sure it will always have this ID?

Reply:
It's the id of the role we want to display. There is no system of role-based videos only HL at this point so its hacky but might just be a one off.